src/manage.c

```diff
@@ -6223,6 +6223,7 @@ get_system_report_types (const char *required_type, gchar ***start,
       if (slave == 0)
         return 2;
 
+      slave_type = scanner_type (slave);
```

Comment:
Build is failing because it's being initialised twice.

Reply:
I've removed the redundant init before this in 6c22de6f528caac56dcb97d1deb7be3de6a70009.